src/services/data_model_engine.js

```diff
@@ -204,8 +204,17 @@ export default class DataModelEngine {
     if (!bundle) {
       throw new Error('Could not fetch the bundle from the shelterer');
     }
-    await this.entityBuilder.validateBundle(bundle);
+    this.entityBuilder.validateBundle(bundle);
+    await this.verifyBundle(bundle);
     await this.entityRepository.storeBundle(bundle);
     return bundle;
   }
+
+  async verifyBundle(bundle) {
+    const bundleSizeLimit = await this.contractManager.configWrapper.bundleSizeLimit();
```

Comment:
Why using wrappers outside repositories?

Reply:
Btw we have it in hermes worker too

Reply:
Config contract has only getters. Not sure if need another class just because we want name like `configRepository` instead of `configWrapper`. We probably should forward all call to wrappers, through repositories, but in this particular case I'm not so confident about it.